ansible/roles/client_cert_update/meta/main.yml

```diff
@@ -1,12 +1,12 @@
 ---
 galaxy_info:
   author: VM Fusion Dev
-  company: CISA Cyber Assessments
   description: Set up the requirements for the cisagov/client-cert-update Docker image
+  company: CISA Cyber Assessments
```

Comment:
Why are `company` and `description` being explicitly moved out of alphabetical order?

Reply:
That is the order in [cisagov/skeleton-ansible-role](https://github.com/cisagov/skeleton-ansible-role/blob/95681aa71c17c7cf69d4658810547dec13d60ccb/meta/main.yml#L3-L5) and part of this PR is making these roles align more with our independent Ansible roles.

Reply:
I think the order should be fixed in `cisagov/skeleton-ansible-role` then, and can be left alone here.

Reply:
I have removed that group of changes as part of my rebase.